DOCS-6697 removed requirement for alphanumeric characters (#54)
mulesoft/docs-visualizer: modules/ROOT/pages/use-tags-in-visualizer.adoc

> When this property is used, the nodes automatically show the assigned tag(s) in the *View* card of Visualizer. The tag cannot be overridden in the UI and can only be modified in properties. You can still add other tags to this node using the UI, however. All tags are displayed as lowercase.

Reviewer: active
```suggestion
When you use this property, the nodes automatically show the assigned tags in the *View* card of Visualizer. You can add but cannot override tags in the UI; you can modify them only in properties. All letters in tag names are displayed in lowercase.
```
